desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -505,10 +512,10 @@ public void updateStatusLabel() {
                     setStatusLabelWarning();
                 }
             } catch (Exception ex) {
+                executeable = false;
```

Comment:
I think it was, but I can't really back it up. Maybe try using an older version (4.5.x or something) and see if I'm right, and if I was, see what has changed.

That executable flag almost _must_ be on purpose. Otherwise why would it be there at all? But I'm not sure why.